spec/unit/inspector_files_spec.rb

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2013-2015 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of version 3 of the GNU General Public License as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE about this file by physical or electronic mail,
+# you may find current contact information at www.suse.com
+require_relative "spec_helper"
+require File.expand_path('../../../helpers/inspector_files', __FILE__)
+
+describe WriteInspectorFiles do
+  describe ".inspect_system" do
+    it "inspects a system with given IP-adress" do
+      ip_adress = "192.168.121.102"
+      expect(Cheetah).to receive(:run).with(
+        anything(), "inspect", ip_adress, "-n", "machine", :stdout => :capture
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Do not use parentheses for method calls with no arguments.